test/prime generator/prime.chpl

```diff
@@ -0,0 +1,46 @@
+/* Primality Test */
+
+use Time;
+proc isPrime(n:int): bool {
```

Comment:
In case you missed it, you don't have to declare argument types or return types for functions in Chapel if you don't want to.  You are also under no obligation to remove them, either :)